### DISCUSSION TITLE
Provide progress on what is occurring

### DISCUSSION THREAD

**Discussion by brettcannon** [brettcannon/WWBD#29] [Ideas]
This could be done in one of two ways. One is to break the Python code into more fine-grained steps and have the TypeScript code call the individual steps. That gives the extension an opportunity to update the progress.

The other option is to open the Output panel when the extension begins so folks can watch as things happen. That would require reading stdout/stderr dynamically from the running process and then to print it out immediately (see the [`TODO` note on this topic](https://github.com/brettcannon/WWBD/blob/dfa9b228d457c0e7b7c91a0f3f1dbc065adc02ad/src/extension.ts#L227)).

**Comment by PaulFWatts** [comment]
My vote would be to open the output panel.
